src/components/base/BaseChip.unit.js

```diff
@@ -0,0 +1,31 @@
+import BaseChip from "@components/base/BaseChip";
+
+describe("@components/base/BaseChip", () => {
+	it("exports a valid component", () => {
+		expect(BaseChip).toBeAComponent();
+	});
+
+	it("renders its props", () => {
+		const wrapper = mount(BaseChip);
+		expect(wrapper.props("size")).toBe("medium");
+		expect(wrapper.props("selected")).toBe(false);
+	});
+
+	it("renders default when not active", () => {
```

Comment:
```suggestion
	it("renders default style when not active", () => {
```
Keep in mind that when running the tests you and one test fails you mostly only read the title and therefore it should be as easy to understand whats wrong as possible.